js/kunai/ui/treeview.js

```diff
@@ -316,6 +316,12 @@ class Treeview {
       }
 
       if (ids.length > 1) {
+        if (this.page_idx.id.type === 'article' && this.page_idx.id.indexes.length > 1) {
+          const lang = $(`[data-lang-id="C++${this.page_idx.cpp_version}"]`)
+          const article = [...lang.find('li')].find(li => $(li).first().text() === this.page_idx.name)
```

Comment:
```suggestion
          const article = Array.from(document.querySelectorAll('[data-lang-id="C++17"] li.article')).find(d => d.innerText === this.page_idx.name)
```

jQueryの書き方を忘れたので素で書きますが、セレクターの絞り込みは1回にできる気がします。

Reply:
レビュー、ありがとうございます。
その方が分かりやすいですね。
参考にさせていただいて、修正しました。